spec/support/vcr_cassettes/debts/get_letters_count_only.yml

```diff
@@ -0,0 +1,37 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://fwdproxy-dev.vfs.va.gov:4465/api/v1/digital-services/debt-letter/get
```

Comment:
Please assign the a variable to the url, https://fwdproxy-dev.vfs.va.gov:4465, in the [vcr support file ](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/vcr.rb)